pkg/configuration/parameters.go

```diff
@@ -168,9 +167,10 @@ func (param *JiraAPIResourceParameters) Parse() {
 		param.Meta.parsed = flag.Parsed()
 	}
 
-	param.InitializeAndValidatePostParse(contextString, issueListString)
+	return contextString, issueListString
 }
 
+// TODO: ... A few custom operation are performed afterward such as initialization and validation.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comment on exported method JiraAPIResourceParameters.InitializeAndValidatePostParse should be of the form "InitializeAndValidatePostParse ..."](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5050785)